one-app-statics/README.md

```diff
@@ -0,0 +1,66 @@
+<h1 align="center">
+  <img src='https://github.com/americanexpress/one-app/raw/master/one-app.png' alt="One App - One Amex" width='50%'/>
+</h1>
+
+# One App Client Assets and CDN
+
+## 🚨 Important 🚨  
+
+This Package is not meant to be installed using npm, it only serves as the CDN for the static client assets for `one-app`,
+if you are looking for `one-app` follow this link: [https://github.com/americanexpress/one-app](https://github.com/americanexpress/one-app)
+
+
+## 👩‍💻 Hiring 👨‍💻
+
+Want to get paid for your contributions to `one-app`?
+> Send your resume to oneamex.careers@aexp.com
+
+## 📖 Table of Contents
+
+* [Features](#-features)
+* [Usage](#-usage)
+* [License](#-license)
+* [Code Of Conduct](#-code-of-conduct)
+* [Contributing](#-contributing)
+
+## ✨ Features
+
+- Improved performance by serving the one app client static assets from a CDN
+- Enhanced security with [Subresource Integrity](https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity)
+
+## 🤹‍ Usage with JSDelivr and Docker
+
+Static assets are created for the correct docker image version in Docker Hub:
+
+[//]: # (TODO: point at the live docker image once pushed)
+[One App Production Docker Image](https://hub.docker.com/r/infoxicator/one-app-test) 
```

Comment:
Is it easier to point at what we know the image will be named now? That way you dont have to go back and edit this later.

Reply:
Will try to get the name from @nellyk